### PR TITLE
Update deployment to run vpp in a separate daemonset

### PR DESCRIPTION
**Pull request by AloysAugustin**

### CODE REVIEW

File: yaml/overlays/test-vagrant/config.yaml [hunk 1]
```diff
@@ -11,8 +11,18 @@ spec:
           env:
             - name: CALICOVPP_DEFAULT_GW
               value: "192.168.0.1"
-        - name: calico-node
-          env:
-            - name: CALICO_IPV4POOL_IPIP
-              value: Always
+# ---
```
Comment (sknat) [inline yaml/overlays/test-vagrant/config.yaml:14]: Some comments stayed :D

File: yaml/overlays/eks/eks-config.yaml [hunk 2]
```diff
@@ -9,18 +9,4 @@ data:  # Configuration template for VPP in EKS
...
-  name: calico-vpp-node
-  namespace: kube-system
-spec:
-  template:
-    spec:
-      containers:
-        - name: vpp
-          resources:
-            limits:
-              hugepages-2Mi: 0
```
Comment (sknat) [inline yaml/overlays/eks/eks-config.yaml:25]: Hmm, not sure if this is really needed for aws. Will check in #147 anyway

**Comment by sknat** [comment]
Appart from nitfixes, looks ok with the fixes on vppdev.sh

**Review by sknat — APPROVED** [review]

File: yaml/overlays/eks/eks-config.yaml (hunk 2 above)
Comment (AloysAugustin) [reply yaml/overlays/eks/eks-config.yaml:25]: The base yaml this kustomization is based on doesn't include hugepages anymore, so this shouldn't change anything.

**Pull request merged**